### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by meg-huggingface**
This does a major "clean up" pass through the code
- Removing functions and variables that were either being duplicated or that we weren't using anymore.
- Removing TODO notes we'd addressed or solved in meetings
- Reorganizing the ordering of some of the variables and functions to make them more coherent chunks
- Adds more commenting through.
This *massively* cuts down the size of app.py, and streamlines it to make it super easy for anyone to add new widgets.

### CODE REVIEW

File: data_measurements/dataset_statistics.py [hunk 1]
```diff
@@ -431,6 +381,7 @@ def load_or_prepare_labels(self, load_only=False):
         or else uses what's available in the cache.
         Currently supports Datasets with just one label column.
         """
+        print(self.dset)
```
Comment (sashavor) [inline data_measurements/dataset_statistics.py:384]: Remove print statement?

Comment (sashavor) [reply data_measurements/dataset_statistics.py:384]: In general, are we logging using `logger`?

File: utils/__init__.py [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
     log_fid = Path(fid).stem
     logs = logging.getLogger(log_fid)
-    logs.setLevel(logging.INFO)
+    logs.setLevel(logging.DEBUG)
     logs.propagate = False
 
     # Logging info to log file
     file_path = ("./log_files/%s.log" % log_fid)
-    print("Logging output in %s " % file_path)
+    print("Logging output in %s" % file_path)
```
Comment (sashavor) [inline utils/__init__.py:14]: ditto- print or log?

**Review by sashavor — APPROVED** [review]
@meg-huggingface and I walked through this together during today's meeting; it all looks good, except maybe a distiction to be made between `print()` and `log()` - we should be logging all around, right?

@TristanThrush , I'm not as familiar with the CLI elements, do you want to look over that?

File: data_measurements/dataset_statistics.py (hunk 1 above)
Comment (meg-huggingface) [reply data_measurements/dataset_statistics.py:384]: Done, and yes.

**Review by TristanThrush — APPROVED** [review]
Couldnt find issues with the CLI stuff. LGTM!

**Pull request merged**